domainfile.json.example

```diff
@@ -0,0 +1,18 @@
+{
+	"bat0": "dom0-gw",
+	"bat10": "dom10-gw",
+	"bat11": "dom11-gw",
+	"bat12": "dom12-gw",
+	"bat13": "dom13-gw",
+	"bat14": "dom14-gw",
+	"bat15": "dom15-gw",
+	"bat16": "dom16-gw",
+	"bat17": "dom17-gw",
+	"bat18": "dom18-gw",
+	"bat19": "dom19-gw",
+	"bat20": "dom20-gw",
+	"bat21": "dom21-gw",
+	"bat22": "dom22-gw",
+	"bat23": "dom23-gw",
+	"bat99": "dom99-gw"
```

Comment:
What's up with the -gw suffix? Shouldn't that match the domain_code given in the Gluon domain?

Reply:
That text can, but does not necessarily need to reflect the content of gluons domain_code.
Though for hanover it does, this format reflects, what @margau has currently running, if I'm not wrong.  However we can change this, as it's just the example of how to format the file.

Reply:
With FFFFM, we actually have different Names per L2-Domains - some on the node-side, mostly PLZ-based, the technical domain ("dom2"), and another one for the gateway, to make the meshviewer-Domain-Name more meaningful.
Example of our meshviewer config.json:
```json
{
      'domain': 'dom17-gw',
      'name': 'Main-Taunus (Gateway)'
    },
{
      'domain': 'ffffm_65817',
      'name': '65817 Eppstein'
    },
{
      'domain': 'ffffm_65812',
      'name': '65812 Bad Soden am Taunus'
    },

```
Both are in the same L2-Domain, but we have more granular Domain Names (per postal code) with only a single gateway instance.